rfcs/0010-postmortem-principles.md

```diff
@@ -0,0 +1,57 @@
+---
+created-on: 2021-04-27
+freshness: 2021-04-27
+owner: @maxneuvians
+status: proposal
+tags:
+  - postmortem
+  - incident response
+---
+
+[La version française suit.](#résumé)
+
+# CDS Principles for running incident response and doing postmortems
+
+## Summary
+Things go wrong. Software breaks. Partnerships aren’t smooth. Delivery takes longer than expected. Sometimes, things go right. In all cases, we should take the time to look back on what happened, how it happened and what we can do better in the future.
+
+Postmortems are a tool for determining what happened, documenting the timelines and discovering what went right and what went wrong. They help us understand how a given outcome occurred and, if necessary, provide the action items to ensure that outcome doesn’t happen again in the future.
```

Comment:
Could we talk about how postmortems, when adopted properly, strengthen the services delivered by that org?

Reply:
Thank you!